docs/Welcome.md

```diff
@@ -40,6 +40,11 @@
 1. [Basics](./ip-allow-listing/Basics.md)
 2. [FAQs](./ip-allow-listing/FAQs.md)
 
+## ISA Declarations

```

Comment:
Is it worth putting ISA declarations and ISA subscriptions under one 'ISA management' header or something? Feels like they're heavily related, but maybe that's just me.

Reply:
I think longer term yes. I think the whole site needs a revisit at some point.